build.sbt

```diff
@@ -16,11 +16,10 @@ def parserCombinators(scalaVersion: String) = "org.scala-lang.modules" %% "scala
   }
 }
 
+val previousVersion: Option[String] = Some("2.0.1")
```

Comment:
2.0.0 was not published: https://github.com/playframework/twirl/releases/tag/2.0.0
(because sbt-dynver read the wrong tag on a commit that had two tags...)